tools/src/bin/sha256tree-benching.rs

```diff
@@ -130,19 +136,20 @@ fn time_per_cons_for_list(
         let cost = red.0;
         let result_2 = node_to_bytes(a, red.1).expect("should work");
         assert_eq!(result_1, result_2);
-
         let duration = start.elapsed().as_nanos() as f64;
+        let duration = (duration - (3.0 * bytes32_clvm_time)) / 2.0;
+        let cost = (cost as f64 - (3.0 * bytes32_clvm_cost)) / 2.0;
```

Comment:
same here, I would expect this program to compare the *cost* of the native operator to the CLVM implementation. But here you alter the cost.